ocs_ci/deployment/deployment.py

```diff
@@ -669,19 +668,33 @@ def deploy_ocs_via_operator(self, image=None):
 
     def deployment_with_ui(self):
         """
-        This method will deploy OCS with openshift-console UI test.
+        Deployment OCS Operator via OpenShift Console
+
         """
-        logger.info("Deployment of OCS will be done by openshift-console")
-        ocp_console = OpenshiftConsole(
-            config.DEPLOYMENT.get("deployment_browser", constants.CHROME_BROWSER)
-        )
-        live_deploy = "1" if config.DEPLOYMENT.get("live_deployment") else "0"
-        env_vars = {
-            "OCS_LIVE": live_deploy,
-        }
-        ocp_console.run_openshift_console(
-            suite="ceph-storage-install", env_vars=env_vars, log_suffix="ui-deployment"
-        )
+        setup_ui = login_ui()
+        deployment_obj = DeploymentUI(setup_ui)
+
+        if config.DEPLOYMENT.get("local_storage"):
+            deployment_obj.mode = "lso"
+        else:
+            deployment_obj.mode = "internal"
+
+        if config.ENV_DATA["platform"].lower() == constants.VSPHERE_PLATFORM:
+            deployment_obj.storage_class_type = "thin_sc"
+        elif config.ENV_DATA["platform"].lower() == constants.AWS_PLATFORM:
+            deployment_obj.storage_class_type = "gp2_sc"
+
+        device_size = str(config.ENV_DATA.get("device_size"))
+        if device_size in ("512", "2048", "4096"):
```

Comment:
We may want to consider adding a constant for `UI_DEVICE_SIZES`. In the event we need to know this list in other places in the code and if those options change in the future it will be better if we only have to update one variable.